From 72edfbc60097158b3d9895ac2b463a95d63c7e72 Mon Sep 17 00:00:00 2001
From: Guhan Balasubramanian <gubalasu@cisco.com>
Date: Fri, 15 Aug 2025 08:31:29 -0700
Subject: [PATCH] ARM: dts: aspeed: morgan800cc bmc mdio support

Enable mdio support for morgan800c bmc to interact with OOB switch
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-morgan800cc.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-morgan800cc.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-morgan800cc.dts
index 7bfb61dd0..6cb5bae6e 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-morgan800cc.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-morgan800cc.dts
@@ -74,3 +74,7 @@ &kcs3 {
 	status = "okay";
 	aspeed,lpc-io-reg = <0xca2>;
 };
+
+&mdio0 {
+	status = "okay";
+};
\ No newline at end of file
-- 
2.43.0

